fix(helm): add 'v' prefix to Helm download URLs

vvs82 opened the pull request:
There is an issue with the current URL templates used for Helm downloads in this registry.

The registry currently points to:
```
https://get.helm.sh/helm-<version>-<os>-<arch>.tar.gz
```
However, Helm officially publishes release artifacts with a v prefix in the version string:
Example (works):
```
https://get.helm.sh/helm-v3.13.3-linux-amd64.tar.gz.sha256sum
```
Current URL (fails with 404):
```
https://get.helm.sh/helm-3.13.3-linux-amd64.tar.gz.sha256sum
```
This PR updates both the archive and checksum URLs to use the correct format with v{{.Version}}, allowing Helm installations via aqua/mise to work properly.

vvs82 committed:
fix(helm): add 'v' prefix to Helm download URLs

suzuki-shunsuke commented:
I think you should add `v` to the package version.

e.g.

https://github.com/aquaproj/aqua-registry/blob/71ba9d464d6c51cd8896d732550e323cf3f3a4a0/pkgs/helm/helm/pkg.yaml#L1-L6

vvs82 commented:
> Please add `v` to the package version.
> 
> e.g.
> 
> https://github.com/aquaproj/aqua-registry/blob/71ba9d464d6c51cd8896d732550e323cf3f3a4a0/pkgs/helm/helm/pkg.yaml#L1-L6

Thanks, Suzuki, I have fixed it by updating my `.mise.toml` file with:
```
[tools]
helm = "v3.13.3"
```
just added `v` symbol, despite:
```
➜  mise ls-remote helm | grep 3.13.3
3.13.3
```
return the version without `v`.
